### DISCUSSION TITLE
Bundle material icons and Roboto fonts in theme nord?

### DISCUSSION THREAD

**Discussion by songhn233** [Milkdown/milkdown#121] [General]
**Describe the bug**

`@milkdown/plugin-slash` README should add instructions for introducing font icons, such as

```html
<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
<link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" />
```

**Comment by Saul-Mirone** [comment]
The icon used in plugin is decided by theme with the `slots.icon` config, so plugin users shouldn't care about the icon set config, it's only decided by the theme.

**Comment by songhn233** [comment]
Thanks, but the default theme `theme-nord` does not import material assets although it specifies the icon class, which may need to be enhanced?

**Comment by Saul-Mirone** [comment]
Yeah, I'm still finding a good way to pack roboto and material icon.
In fact, I think maybe it's better to have users to bundle it themselves, because if we package icons and fonts, what if users want to use roboto or material fonts in other parts of their app? That's will cause the fonts be packed twice.
